### ISSUE TITLE
Misschien is het leuk om een foto van de kippensoep toe te voegen?

### ISSUE THREAD

**Issue by serra** [PoeticDust/PoeticDust.github.io#2]

**Comment by PoeticDust** [comment]
Ja, was ik al van plan. Allen geeft GitHub de hele tijd een error (404) als ik op de knop "Recepten" klik.
Ik snap niet waarom GitHub de pagina niet kan vinden, zie ik misschien iets over het hoofd?

**Comment by serra** [comment]
https://github.com/PoeticDust/PoeticDust.github.io/blob/master/index.html#L23

Daar staat `Recepten.html` maar het bestand heet `recepten.html` (hoofdletter R). Op Windows computers maakt dat niet uit, maar op (bijna alle) andere computers wel. Dat is het denk ik.
